src/main/java/com/google/api/generator/spring/composer/SpringPropertiesClassComposer.java

```diff
@@ -331,7 +307,18 @@ private static Map<String, TypeNode> createDynamicTypes(Service service, String
                 .setName(Utils.getServicePropertiesClassName(service))
                 .setPakkage(packageName)
                 .build());
+
+    // TODO: This should move to static types once class is added into spring-cloud-gcp-core
```

Comment:
@zhumin8 On a second thought, do you think it would be easier to leave this as a dynamic type in generator code until the shared classes are published and available in maven central?  

Let me know if there's anything I'm missing here - currently, this use of dynamic type unblocks code generation, and the generated code is able to compile against a locally installed snapshot jar of spring-cloud-gcp.

Reply:
I think we can leave as dynamic type temporarily and unblock. 
But we do need to switch to local repo for spring-cloud-gcp dependencies eventually to develop on the snapshot version.

> currently, this use of dynamic type unblocks code generation, and the generated code is able to compile against a locally installed snapshot jar of spring-cloud-gcp.

yes. But same argument on adding concrete types for other spring denpendencies, it's safer to have the imported class confirmed in the generator than validate via compile in generated module.

Reply:
Gotcha, that makes sense. Thank you for the explanation!